modules/nf-core/mitohifi/findmitoreference/main.nf

```diff
@@ -30,7 +30,15 @@ process MITOHIFI_FINDMITOREFERENCE {
 
      cat <<-END_VERSIONS > versions.yml
     "${task.process}":
-        mitohifi: \$( mitohifi.py --version )
+        mitohifi: \$( mitohifi.py --version 2>&1 | head -n1 | sed 's/^.*MitoHiFi //; s/ .*\$//' )
+    END_VERSIONS
+    """
+
+    stub:
+    """
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        mitohifi: \$( mitohifi.py --version 2>&1 | head -n1 | sed 's/^.*MitoHiFi //; s/ .*\$//' )
```

Comment:
```suggestion
    stub:
    """
    touch ${prefix}.fasta
    touch ${prefix}.gb

    cat <<-END_VERSIONS > versions.yml
    "${task.process}":
        mitohifi: \$( mitohifi.py --version 2>&1 | head -n1 | sed 's/^.*MitoHiFi //; s/ .*\$//' )
```

The goal of a stub run is to emulate the actual process so the input and output files should be as close as possible to the real input and output. Creating empty files is sufficient in this case. (The stub will throw an error if the fasta and gb files aren't there)

Reply:
Aaah, that makes sense! Thanks!

Reply:
Oh you should also add the creation of the `prefix` variable to the stub section :)

Reply:
Oh right I forgot about that, my bad 😁. You can just use dummy file names in this case :)

Reply:
No worries, I'll do that! :-)